src/test/java/racingcar/CarTest.java

```diff
@@ -16,7 +16,7 @@ public class CarTest {
 	@DisplayName("자동차 이름 5자 초과하면 예외발생")
 	void carNameOverFive() {
 		assertThatThrownBy(() -> new Car("abcdef"))
-			.isInstanceOf(RuntimeException.class);
+			.isInstanceOf(IllegalArgumentException.class);
```

Comment:
메시지까지 검증한다면 테스트에 대한 신뢰도를 높일 수 있을 것 같아요 😃 
```suggestion
			.isInstanceOf(IllegalArgumentException.class)
			.hasMessage("~~");
```